packages/gatsby-theme-docs/gatsby-ssr.js

```diff
@@ -94,5 +95,16 @@ export const replaceRenderer = ({
         __html: css,
       }}
     />,
+    <style
```

Comment:
I could have put that into the global emotion style too but I think it's better to handle the topic in the same place as other such purely technical hacks.